cfg/admin.txt

```diff
@@ -1 +1 @@
-jayehh role=admin
+jayehh role=admin
```

Comment:
This file shouldn't be different at all, what the heck. Where are the merge conflicts coming from? Help.